app/assets/javascripts/components/locales/fr.jsx

```diff
@@ -36,7 +36,6 @@ const fr = {
   "compose_form.publish": "Pouet",
   "compose_form.sensitive": "Marquer le média comme délicat",
   "compose_form.spoiler": "Masquer le texte derrière un avertissement",
```

Comment:
I changed compose_form.spoiler_placeholder meaning. so translation should change.